server/src/main/java/io/crate/metadata/doc/DocIndexMetadata.java

```diff
@@ -692,21 +692,26 @@ public Settings tableParameters() {
                 || (columnDataType.id() == ArrayType.ID
                     && ((ArrayType) columnDataType).innerType().id() == ObjectType.ID)) {
                 if (columnProperties.get("properties") != null) {
-                    builder.putAll(getAnalyzers(newIdent, (Map<String, Object>) columnProperties.get("properties")));
```

Comment:
I think we could extend the `MapBuilder` with a `putAll` and keep this as it was before.